.github/workflows/pr-check.yml

```
@@ -0,0 +1,59 @@
+      run: npm ci
+      
+    - name: Run all tests
+      run: npm test
+      
+    - name: Test server startup
+      run: |
+        # Test that the server can start without errors
+        timeout 5s node index.js &
+        sleep 2
+        kill %1 || true
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)